### ISSUE TITLE
Excel javascript api fails to read range from sheet Edge vs IE

### ISSUE THREAD

**Issue by IvandeJong** [OfficeDev/office-js#703]
We have an excel addin which writes an range to an excel sheet and lets user modify it. Once he is done he will press a button on wich we will read the range from the sheet and will try to parse it.

Excel has recently updated its integration browser:
https://docs.microsoft.com/en-us/office/dev/add-ins/concepts/browsers-used-by-office-web-add-ins

Windows 10 ver. < 1903 / Office 365
Internet Explorer 11

Windows 10 ver. >= 1903 / Office 365 ver < 16.0.11629
Internet Explorer 11

Windows 10 ver. >= 1903 / Office 365 ver >= 16.0.11629
Microsoft Edge*

We traced a problem with this edge based implementation. and made a code example for this.

When we load an excel sheet with a range from A1:A100 with random strings (80 karakters each) we fail to load the range from the worksheet.
The task pane in the ie version does hit the comment but in edge it does not get hit.

We try to load the range with the folowing code:
```javascript
// Run a batch operation against the Excel object model
Excel.run(function (ctx) {
    // Create a proxy object for the used range and load its properties    
    var sourceRange = ctx.workbook.worksheets.getActiveWorksheet().getUsedRange().load("values, rowCount, columnCount");
    // Run the queued-up command, and return a promise to indicate task completion
    return ctx.sync()
                .then(function () {
                    // Unhit code in edge only ????              
                })
                .then(ctx.sync);
        })
        .catch(errorHandler);
```

[Example task pane](https://github.com/OfficeDev/office-js/files/3573367/Home.zip)

For the time being could you sugest a workaround?

**Comment by IvandeJong** [comment]
Duplicate of https://github.com/OfficeDev/office-js/issues/697
